app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -40,10 +40,10 @@ expander:
 inset_text:
   loan-increase:
     text: |-
-      From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses have increased. This means the amount you can apply to borrow to cover tuition fees has also increased. The increased amounts are shown on this page.
+      For the <%= v :finance_academicyear %>, the maximum fees that providers can charge for postgraduate teacher training courses have increased. This means the amount you can apply to borrow to cover tuition fees has also increased. The increased amounts are shown on this page.
```

Comment:
This (and the other banner) hasn't worked:
![image](https://github.com/user-attachments/assets/863dd09c-9e78-4453-a726-6a1befa8b7eb)

Reply:
However, I'm not sure this is the right approach

The banners will be deleted before next cycle - they are temporary. Once this happens, there will be no reference to academic year on the page at all.

I would maybe suggest adding a new line somewhere between line 49 and 64 that will be permanently on the page - and then leaving the banners as is for now?